docs/developer/getting-started.md

```diff
@@ -24,7 +26,8 @@ Follow the readme instructions in the root of the repository.
 
 ## Databases
 
-Standalone database snapshots are currently unavailable, please use the [development virtual machine](/developer/vm.html).
+Please contact the intake24 team for the latest development virtual machine. [Please also check the guidance on using the development virtual machine](https://docs.intake24.org/developer/vm.html).
+Alternatively, obtain database snapshots from the intake24 team and import them into a database browser (e.g. DBeaver). Please check the guidance on [importing database snapshots with DBeaver](https://dbeaver.com/docs/wiki/Backup-Restore/). If using the database snapshots, you need to set up your own Redis instance.
```

Comment:
Redis is needed regardless whether you use DB snapshots. I think last sentence can be droppped.